apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts

```diff
@@ -75,7 +75,7 @@ export class Webhook {
       identifier: messageIdentifier,
       _environmentId: command.environmentId,
       _organizationId: command.organizationId,
-    });
+    } as any);
```

Comment:
So this one won't cache the request at the moment because it lacks the subscriberId, we can make add extra logic where we cache it by the `identifier` the question is do we want to?